nltk/parse/transitionparser.py

```diff
@@ -490,7 +490,7 @@ def _create_training_examples_arc_eager(self, depgraphs, input_file):
         print(" Number of valid (projective) examples : " + str(countProj))
         return training_seq
 
-    def train(self, depgraphs, modelfile):
+    def train(self, depgraphs, modelfile, verbose=True):
```

Comment:
Thanks for your response @fievelk. I notice that there are about 120 `verbose=False` in our codebase, and about 10 `verbose=True`. But since you're preserving existing behaviour, I'm happy to let it stand and merge this, and deal with this inconsistency in a separate issue: https://github.com/nltk/nltk/issues/1509